types/cavy/index.d.ts

```diff
@@ -53,6 +53,7 @@ export class TestScope {
     beforeEach(fn: () => void): void;
     press(identifier: string): Promise<void>;
     fillIn(identifier: string, str: string): Promise<void>;
+    focus(identifier: string): Promise<void>;
```

Comment:
As this is a new api, can we amend the inline doc, like that one from the upstream change?
```text
Returns a promise, use await when calling this function.
Promise will berejected if the component is not found.
```